src/backend/joanie/core/api/client.py

```diff
@@ -1380,3 +1381,70 @@ def preview_template(self, request, pk=None):  # pylint: disable=invalid-name, u
             as_attachment=True,
             filename="contract_definition_preview_template.pdf",
         )
+
+
+class NestedCourseOrderViewSet(NestedGenericViewSet, mixins.ListModelMixin):
```

Comment:
This is related to `learner` (I think this name is not relevant) and you call it `CourseOrder`. It's not easy to understand. Reading it I expect to have an entire `order`. Not only thinkgs related to the owner. IMO this endpoint should return everything and `owner` sideloads the user.